CONTRIBUTING.md

```diff
@@ -78,6 +78,12 @@ The following dependencies are also needed for testing:
 pip install typing pytest scipy numpy parameterized
 ```
 
+Optional packages to install if you want to run related tests:
+
+```bash
+pip install librosa requests soundfile kaldi_io transformers
```

Comment:
Installation of `librosa` is sometimes tricky. 

https://github.com/pytorch/audio/blob/380800ce09ef9b31475d298eb3e60f096056caca/.circleci/unittest/linux/scripts/install.sh#L55

Instead of giving the instruction with `pip install`, can you just simply list these libraries?
Something like

- `librosa`
- `requests`
- `soundfile`
- `kaldi_io`
- `transformers`
- `fairseq` (it has to be newer than `0.10.2`, so you will need to install from source. Commit `e6eddd80` is known to work.)

Reply:
I've added these [here](https://github.com/pytorch/audio/pull/1584/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R81)